src/raspbian/10/helix/arm32v6/Dockerfile

```diff
@@ -1,5 +1,5 @@
-# dummy FROM to establish a dependency
-FROM mcr.microsoft.com/dotnet-buildtools/prereqs:raspbian-10-crossdeps AS dependency
+# dummy F R O M to establish a dependency
+# FROM mcr.microsoft.com/dotnet-buildtools/prereqs:raspbian-10-crossdeps
```

Comment:
Why is the comment needed? I wouldn't want to depend on us continuing to parse comments.